install

```diff
@@ -226,24 +225,24 @@ end
 
 wait_for_user if STDIN.tty? && !ENV["TRAVIS"]
 
-if File.directory? HOMEBREW_PREFIX
-  sudo "/bin/chmod", "g+rwx", *group_chmods unless group_chmods.empty?
-  sudo "/bin/chmod", "u+rwx", *user_chmods unless user_chmods.empty?
-  sudo "/usr/sbin/chown", ENV["USER"], *chowns unless chowns.empty?
-  sudo "/usr/bin/chgrp", "admin", *chgrps unless chgrps.empty?
```

Comment:
Not sure why you've moved these; no need to run them if `HOMEBREW_PREFIX` didn't exist.